x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -95,7 +96,14 @@ export const setSignalsStatusRoute = (
           if ('signal_ids' in request.body) {
             const { signal_ids: signalIds } = request.body;
```

Comment:
If alert gets reopened, what should happen to `workflow.reason`?
How can it be reset? Won't it be misleading for re-opened alert to have this reason to be the same when it was closed?

Reply:
Good point,  @paulewing what do you think?

Reply:
Good point here, let's talk to @paulewing on Slack